queries/yaml/highlights.scm

```diff
@@ -13,17 +13,36 @@
 (tag) @type
 (yaml_directive) @keyword
 (ERROR) @error
+
+(block_mapping_pair key: (_) @field)
+(block_mapping_pair
+  key: (flow_node [(double_quote_scalar) (single_quote_scalar)] @field))
+(block_mapping_pair
+  key: (flow_node (plain_scalar (string_scalar) @field)))
+
+(flow_mapping key: (_) @field)
+(flow_mapping
+  (_ key: (flow_node [(double_quote_scalar) (single_quote_scalar)] @field)))
+(flow_mapping
+  (_ key: (flow_node (plain_scalar (string_scalar) @field))))
```

Comment:
so many nested stuff...

Reply:
Du you really mean to capture that key multiple times? Because that's what will end up happening here.

Reply:
I'm not sure if I understood your comment, do you mean the previous capture of `(_) @field`? If so, we still need more specific captures for `(string_scalar) (double_quote_scalar) (single_quote_scalar)` since they are captured as strings below (ts gives precedence to specific capture groups).

Reply:
you can check this by removing these queries and leaving only `(_) @field`